aptos-move/framework/aptos-token/sources/property_map.move

```diff
@@ -12,7 +12,7 @@ module aptos_token::property_map {
     const EKEY_COUNT_NOT_MATCH_VALUE_COUNT: u64 = 4;
     const EKEY_COUNT_NOT_MATCH_TYPE_COUNT: u64 = 5;
 
-    struct PropertyMap has drop, store {
+    struct PropertyMap has copy, drop, store {
```

Comment:
Not necessarily need to have a copy here but its your choice. If you don't have copy here, you can still easily define a `clone` function using the inner copyable things.